screen/store/components/pages/AccountPage/AccountPage.js

```diff
@@ -41,50 +48,160 @@ var AccountPage = {
       }.bind(this));
     },
     addCustomerAddress() {
```

Comment:
Could we have more granular error reporting than this?
```
var errors = [];
if(this.customerAddress.countryGeoId == null ||
    this.customerAddress.countryGeoId.trim() == "") {
    errors.push("Please select a country");
}


if(this.customerAddress.city == null ||
    this.customerAddress.city.trim() == "") {
    errors.push("Please select a city");
}

if(errors.length > 0) {
    this.responseMessage = errors.join("<br/>");
    return;
}
```
Is "toName" a required field?